### DISCUSSION TITLE
occlum-gcc: command not found

### DISCUSSION THREAD

**Discussion by lili8808** [occlum/occlum#993] [Q&A]
I have installed Occlum using this link :https://github.com/occlum/occlum/blob/master/docs/install_occlum_packages.md 
then I run the command : `cd /tmp && wget https://raw.githubusercontent.com/occlum/occlum/master/demos/hello_c/hello_world.c `
and then when I run `occlum-gcc -o hello_world hello_world.c `
it shows me error:
`occlum-gcc: command not found`
Does any body know the reason and solution?
Thanks

**Comment by tychobrailleur** [comment]
Have you sourced `/etc/profile` in your current terminal?   `occlum-gcc` should be under `/usr/local/occlum/bin`, and added to the path by `/etc/profile.d/occlum-gcc.sh`.
